test/project_types/rails/commands/create_test.rb

```diff
@@ -199,6 +205,19 @@ def perform_command(add_cmd = nil)
                              --shop_domain=testshop.myshopify.com)
         run_cmd(default_new_cmd << add_cmd, false)
       end
+
+      def create_gem_path_and_binaries
+        FileUtils.mkdir_p('gem/path/bin')
+        gem_path = File.expand_path('gem/path')
+        ['bundle', 'rails', 'spring'].each do |f|
```

Comment:
Nothing wrong here, but I'm surprised rubocop let this through - I thought it complained about string arrays not being `%w()`.